ros2topic/ros2topic/verb/echo.py

```diff
@@ -103,6 +104,8 @@ def add_arguments(self, parser, cli_name):
                                                  'as well as m (the message).')
         parser.add_argument(
             '--once', action='store_true', help='Print the first message received and then exit.')
+        parser.add_argument(
+            '--time-out', type=positive_float, help='Set a timeout for waiting')
```

Comment:
I see the option `--timeout` in daemon, not hyphen separated between. probably use the same format?

```suggestion
            '--timeout', metavar='N', type=positive_float, help='Set a timeout in seconds for waiting')
```

probably set the default? what happens if user issues `ros2 topic echo --timeout /chatter`?

Reply:
Thanks! Addressed in ecde453 and cc58815